cc3d/twedit5/Plugins/CC3DProject/NewSimulationWizard.py

```diff
@@ -894,6 +894,12 @@ def validateCurrentPage(self):
                                     QMessageBox.Ok)
                 return False
 
+            elif not name.isidentifier():
+                QMessageBox.warning(self, "Invalid simulation name",
+                                    "Please specify a name with no spaces that begins with a letter or '_'",
```

Comment:
with no spaces and no ```-```

Reply:
you mean minus sign instead of underscore, right?

Reply:
yes